unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/GoToGenesisPlazaHUD/GoToGenesisPlazaHUDView.cs

```diff
@@ -0,0 +1,49 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCL.GoToGenesisPlazaHUD
+{
+    public class GoToGenesisPlazaHUDView : MonoBehaviour
+    {
+        public bool isOpen { get; private set; } = false;
+
+        public event System.Action OnClose;
+
+        private const string PATH = "GoToGenesisPlazaHUD";
+        private const string VIEW_OBJECT_NAME = "_GoToGenesisPlazaHUD";
+
+        [SerializeField] private ShowHideAnimator GoToGenesisPlazaAnimator;
+        [SerializeField] private Button cancelButton;
+        [SerializeField] internal Button continueButton;
+
+        private void Initialize()
+        {
+            gameObject.name = VIEW_OBJECT_NAME;
+
+            cancelButton.onClick.AddListener(() =>
+            {
+                SetVisibility(false);
+            });
```

Comment:
This logic is controller responsibility, the controller should be suscribing to an `view.OnCancelClick` or similar and toggle the view visibility.

Reply:
You're right. Done! Thanks!